proxy/list.go

```diff
@@ -39,7 +39,7 @@ func ListFunctionsToken(gateway string, tlsInsecure bool, token string, namespac
 	if len(token) > 0 {
 		SetToken(getRequest, token)
 	} else {
-		SetAuth(getRequest, gateway)
+		AddAuth(getRequest, gateway)
```

Comment:
Why was this renamed?

Reply:
It was not renamed but basically we are using a different function which is checks whether it is `oauth2` or `basic` auth and add the corresponding header. I have to update this method to all commands .. I just did this for `faas-cli list`  command.

Reply:
We can also update the `SetAuth` method and we will not have to add the support to each command separately. 

Since it is already being called inside each command but I rather chose to use a separate method `addAuth` and add the support gradually.